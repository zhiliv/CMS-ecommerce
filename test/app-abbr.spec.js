import { shallowMount } from '@vue/test-utils'
import ABBR from '@/components/app/abbr/abbr.vue'

describe('app-abbr.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(ABBR, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple simple-xl',
        styles: 'background: white',
        id: 'indent',
        title: 'Подсказка'
      },

      slots:{
        default: 'Содержимое'
      }
    })
  })
  it('Содержимое html для app-abbr ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
