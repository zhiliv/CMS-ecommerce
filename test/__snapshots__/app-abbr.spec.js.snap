// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-abbr.js Содержимое html для app-abbr  1`] = `
<abbr
  class="initialism simple simple-xl"
  id="indent"
  style="background: white;"
  title="Подсказка"
>
  Содержимое
</abbr>
`;
